tools/clang/tools/dxcompiler/dxcutil.cpp

```diff
@@ -50,25 +50,22 @@ namespace {
 // AssembleToContainer helper functions.
 
 // return true if the internal validator was used, false otherwise
-bool CreateValidator(CComPtr<IDxcValidator> &pValidator,
-                     hlsl::options::ValidatorSelection SelectValidator =
-                         hlsl::options::ValidatorSelection::Auto) {
-  bool bInternal =
-      SelectValidator == hlsl::options::ValidatorSelection::Internal;
-  bool bExternal =
-      SelectValidator == hlsl::options::ValidatorSelection::External;
-  bool bAuto = SelectValidator == hlsl::options::ValidatorSelection::Auto;
-
-  // default behavior uses internal validator, as well as
-  // explicitly specifying internal
-  if (bInternal || bAuto) {
+bool CreateValidator(CComPtr<IDxcValidator> &pValidator) {
+
+  // default behavior uses internal validator
+  // TODO: Check if environment variable is set, if so,
+  // use external validator
```

Comment:
This comment can be deleted and the code can be simplified now, with the else block removed.